src/domain/fiat/display-currency.ts

```diff
@@ -68,3 +68,40 @@ export const getCurrencyMajorExponent = (
     return MajorExponent.STANDARD
   }
 }
+
+export const newDisplayAmountFromNumber = <T extends DisplayCurrency>({
+  amount,
+  currency,
+}: {
+  amount: number
```

Comment:
Why would it every be a number?

Reply:
It's from a part of the code that we haven't refactored away from `number` as yet. All `...Metadata` methods still take display amounts as numbers (branded `DisplayCurrencyBaseAmount` type) so this gets passed around quite a bit in different use-cases.

This could be addressed if we decide to include removing the `DisplayCurrencyBaseAmount` type from the code as part of our post-cleanup, which currently has 150 instances of being used in the codebase.